src/App.js

```diff
@@ -1,42 +1,42 @@
 import "./App.css";
-import logo from "./logo.png";
-
+import { Routes, Route } from "react-router-dom";
+import Mockman from "mockman-js";
+import {  ExplorePage } from "./pages/ExplorePage/ExplorePage";
+import {NavBar} from "./components/NavBar/NavBar";
+import {Footer} from "./components/Footer/Footer";
+import {Aside} from "./components/aside/Aside";
+import { useEffect } from "react";
+import {SingleVideoPage} from "./pages/SingleVideoPlayer/SingleVideoPage";
+import axios from "axios";
 function App() {
+  useEffect(
+       async () => {
+      try {
+        const response = await axios.post("/api/auth/login", {
+              email: "adarshbalika@gmail.com",
+    password: "adarshBalika123",
+        });
+        console.log(response.data.foundUser);
+        localStorage.setItem("token", response.data.encodedToken);
+      } catch (error) {
+        console.log(error);
+      }
+    
+  },[])
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} alt="mockBee logo" width="180" height="180" />
-        <h1 className="brand-title">
-          Welcome to <span>mockBee!</span>
-        </h1>
-        <p className="brand-description">
-          Get started by editing <code>src/App.js</code>
-        </p>
-        <div className="links">
-          <a
-            href="https://mockbee.netlify.app/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Explore mockBee
-          </a>
-          <a
-            href="https://mockbee.netlify.app/docs/api/introduction"
-            target="_blank"
-            rel="noreferrer"
-          >
-            API Documentation
-          </a>
-          <a
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+      <NavBar/>
+      <Aside/>
+      <Routes>
+        <Route path="/" element={<ExplorePage/>}/>
+        <Route path="/singleVideo-page" element={<SingleVideoPage/>}/>
+        <Route path="/singleVideo-page/:source" element={<SingleVideoPage/>}/>
```

Comment:
Here, even if you have a path="/singleVideo-page/:source" will work, You can remove like 32.
You also have not included the routes to other pages.

Reply:
Thanks for reviewing I was about to remove that thanks for noticing it